frontend/src/components/VDetailPerson.vue

```diff
@@ -70,6 +76,24 @@ export default {
 
   components: {
     VTreeView
+  },
+
+  props: {
+    uuid: String,
+    result: Object
+  },
+
+  methods: {
+    getIcon (locationType) {
```

Comment:
similar to `VDetailOrganisation`. We can generalise with a mixin. Let's have chat, I'll show you how to do that.